config/license.js

```diff
@@ -0,0 +1,11 @@
+/*
+Copyright <%= YEAR %> Adobe. All rights reserved.
```

Comment:
We should be able to make this update separately - it's not necessary for the 11ty implementation.

Reply:
Done